src/internal/hooks/useExchangeRate.tsx

```diff
@@ -32,12 +33,15 @@ export async function useExchangeRate({
   const toToken = selectedInputType === 'crypto' ? usdcToken : token;
 
   try {
-    const response = await getSwapQuote({
-      amount: '1', // hardcoded amount of 1 because we only need the exchange rate
-      from: fromToken,
-      to: toToken,
-      useAggregator: false,
-    });
+    const response = await getSwapQuote(
+      {
+        amount: '1', // hardcoded amount of 1 because we only need the exchange rate
+        from: fromToken,
+        to: toToken,
+        useAggregator: false,
+      },
+      RequestContext.Wallet,
```

Comment:
non-blocking question: when should we allow this to be `hook` vs hardcoded to the "default" component in which it's used?